### PR TITLE
Adding new blog for Lucky 0.29

### PR DESCRIPTION
**Pull request by jwoertink**

### CODE REVIEW

File: src/posts/lucky_0.29.cr [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    after all of the `before_save` blocks have completed allowing you to
+    do any necessary setup before hand.
+
+    ```crystal
+    before_save do
+      code.value ||= "1234"
+    end
+
+    default_validations do
+      validat_required code
```
Comment (matthewmcgarvey) [inline src/posts/lucky_0.29.cr]:
```suggestion
      validate_required code
```

**Review by matthewmcgarvey — APPROVED** [review]
Minor typo, otherwise looks good 👍

File: src/posts/lucky_0.29.cr [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  def published_at : Time
+    Time.utc(year: 2021, month: 12, day: 8)
+  end
+
+  def summary : String
+    <<-TEXT
+    The Lucky 0.29 release brings us many more
+    quality of life updates, bug fixes, and a
+    few new features to really make your app
+    solid.
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]: Not that it matters, but why did you wrap to 43 characters?

File: src/posts/lucky_0.29.cr [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    We've done it again! Lucky 0.29.0 is now out, and a few apps
+    in the wild have already upgraded.
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]:
```suggestion
    Lucky 0.29.0 is now out, and a few apps
    in the wild have already upgraded!
```

This made me chuckle, but it feels self-congratulatory in a way that might be off putting. What do you think?

File: src/posts/lucky_0.29.cr [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+    * Lucky no longer writes to the body on HTTP HEAD calls. (oops 😅)
+    * The Lucky ecosystem development dependencies (like Ameba) are no longer installed on every app.
+    * HTML boolean attributes use valid markup. (e.g. `<input checked />` VS `<input checked="true" />` )
+    * The 404 error page no longer returns a 200 response.
+    * You can now define a model named `Status`.
+    * Migrations are sorted in the correct order when using Breeze.
+    * `lucky dev` no longer clashes with Elixir development via the `mix` command.
+
+    ### Better development
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]: Better or worse? What do you think?

```suggestion
    ### Improved developer experience
```

File: src/posts/lucky_0.29.cr [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+    In some instances, you may need to just skip all of the default validations
+    , including the ones Lucky runs for you. To do this, we've added the `skip_default_validations`
+    macro. Place this in the operation you wish to skip those validations.
+
+    ```crystal
+    class SaveCode < Code::SaveOperation
+      skip_default_validations
+
+    end
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]:
```suggestion
    class SaveCode < Code::SaveOperation
      skip_default_validations
    end
```

or 

```suggestion
    class SaveCode < Code::SaveOperation
      skip_default_validations

      # ...
    end
```

File: src/posts/lucky_0.29.cr [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+      settings.i18n_backend = I18n.new
+    end
+    ```
+
+    > [Read the PR](https://github.com/luckyframework/avram/pull/757) for more details.
+
+    ## Parting words
+
+    We want to acknowledge and thank the community members for sticking with the Lucky framework.
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]: Do we want to capitalize `Framework`?

```suggestion
    We want to acknowledge and thank the community members for sticking with the Lucky Framework.
```

File: src/posts/lucky_0.29.cr [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+    Lucky has set a goal to be a solid and stable choice for a Crystal based web-framework and in
+    most cases, we feel we have hit the mark.
+
+    At this point we had hoped to be closer to the 1.0 release, but due to life outside of OpenSource,
```
Comment (edwardloveall) [inline src/posts/lucky_0.29.cr]: We're not very consistent about this but I don't think [open source](https://en.wiktionary.org/wiki/open_source) is a proper noun. Also, confusingly, when it's an [adjective](https://en.wiktionary.org/wiki/open-source) (e.g. open-source software) it should use the hyphen. We've been pretty good about that one so far at least in blog posts.

```suggestion
    At this point we had hoped to be closer to the 1.0 release, but due to life outside of open source,
```

**Review by edwardloveall — APPROVED** [review]
Great work! Only a few minor questions. And I really do mean questions. I think it's pretty clear when I'm saying "you should change this" vs "should we change this?" Please do take them as genuine questions and not veiled suggestions. 🙂

File: src/posts/lucky_0.29.cr (hunk 2 above)
Comment (jwoertink) [reply src/posts/lucky_0.29.cr]: because my font is super zoomed in, and that's the edge 😂

Comment (jwoertink) [reply src/posts/lucky_0.29.cr]: oh wait... this is on the upper part... umm no clue why I did this lol

**Comment by jwoertink** [comment]
Sweet! Thanks for the reviews. I'll merge this in.

**Pull request merged**